libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -943,6 +943,18 @@ void NavEKF2::getVelNED(int8_t instance, Vector3f &vel) const
     }
 }
 
+// return estimate of true airspeed vector in body frame in m/s for the specified instance
+// An out of range instance (eg -1) returns data for the primary instance
+// returns false if estimate is unavailable
+bool NavEKF2::getAirSpdVec(int8_t instance, Vector3f &vel) const
+{
+    if (instance < 0 || instance >= num_cores) instance = primary;
```

Comment:
nitpick: normally we would have these on separate lines..

Reply:
Thats the same format used elsewhere throughout EKF2